Surveys/capture-device-information/capture-device-information.js

```diff
@@ -17,7 +17,7 @@
                 userAgentField.readOnly = true;
                 userAgentField.hidden = true;
                 
-                // Checkbox Frontend uses AngularJS, we need to force a DOM re-render.
+                // Checkbox Frontend uses Angular, we need to force a DOM re-render.
```

Comment:
@impurator @PavelKrysenko Thank you for the previous comment-- CLEARLY I need to update my GitHub notifications to go to my @checkbox.com email :)

I've updated the comment to reflect the correct framework.